x-pack/plugins/session_view/public/components/process_tree_node/buttons.tsx

```diff
@@ -98,8 +104,15 @@ export const AlertButton = ({
       onClick={onToggle}
       data-test-subj="processTreeNodeAlertButton"
     >
-      <FormattedMessage id="xpack.sessionView.alerts" defaultMessage="Alerts" />
-      <EuiIcon css={buttonArrow} size="s" type={getExpandedIcon(isExpanded)} />
+      {alertsCount > 1 ? (
+        <FormattedMessage id="xpack.sessionView.alerts" defaultMessage="Alerts" />
+      ) : (
+        <FormattedMessage id="xpack.sessionView.alert" defaultMessage="Alert" />
+      )}
+      {alertsCount > 1 && (
+        <span css={alertsCountNumber}>({alertsCount > 99 ? '99+' : alertsCount})</span>
```

Comment:
Hopefully this threshold is never reached haha, for one process to have 99 alerts would mean there are at least 30 different rules firing on each life cycle event of the process. 99+ should be, "so i hear you like alert fatigue".